src/App.tsx

```diff
@@ -12,156 +18,20 @@ export const App = () => (
     >
       <div className="container">
         <div className="navbar-brand">
-          <a className="navbar-item" href="#/">
+          <NavLink to="/" className={getLinkClass}>
```

Comment:
You can move all routes to enum. 
Then use this enum in all cases where you need links.